dev/PipsPager/PipsPager.cpp

```diff
@@ -85,12 +85,19 @@ void PipsPager::OnApplyTemplate()
     }(GetTemplateChildT<winrt::Button>(c_nextPageButtonName, *this));
 
     m_pipsPagerElementPreparedRevoker.revoke();
+    m_pipsAreaGettingFocusRevoker.revoke();
+    m_pipsAreaBringIntoViewRequestedRevoker.revoke();
     [this](const winrt::ItemsRepeater repeater)
     {
         m_pipsPagerRepeater.set(repeater);
         if (repeater)
         {
             m_pipsPagerElementPreparedRevoker = repeater.ElementPrepared(winrt::auto_revoke, { this, &PipsPager::OnElementPrepared });
+            m_pipsAreaGettingFocusRevoker = repeater.GettingFocus(winrt::auto_revoke, { this, &PipsPager::OnPipsAreaGettingFocus });
```

Comment:
>GettingFocus [](start = 53, length = 12)

Do winui still support rs1? GettingFocus is introduced by rs2

Reply:
nope, the last version we test for is rs2 :)